frame/support/procedural/src/pallet/expand/call.rs

```diff
@@ -178,7 +178,10 @@ pub fn expand_call(def: &mut Def) -> proc_macro2::TokenStream {
 			#(
 				#( #[doc = #fn_doc] )*
 				#fn_name {
-					#( #args_compact_attr #args_name_stripped: #args_type ),*
+					#(
+						#[allow(missing_docs)]
+						#args_compact_attr #args_name_stripped: #args_type
```

Comment:
This looks a bit excessive to document the fields of the arguments in an extrinsic function. Would the compiler really complain about missing docs otherwise?

Reply:
Yes it complains about the missing. I mean it makes sense, because they are not there :P